Test/IntegrationTests/MqttClient/README.md

```diff
@@ -0,0 +1,21 @@
+#  Mqtt Client Sample
+
+This sample is an iOS application that launches the Mqtt Client and subscribes to the topic `test/topic`.
+
+## How to run the application
+1. Open project `Samples/MqttClient/MqttClient.xcodeproj` with XCode
+2. Signing. Signing is required for iOS application. Setup `Project > Signing & Capabilities > Team` to automatically manage signing.
+3. Update Dependency. The project depends on aws-crt-swift. Go to `File > Add package dependency > Add Local ... ` and select your local aws-crt-swift. Alternatively, you can also enter Package URL `https://github.com/awslabs/aws-crt-swift` for released versions.
```

Comment:
This should depend upon the release version and people can replace that with local version if desired instead of always adding it manually.

Reply:
The sample is already setup to point to local version, so that the CI could test any local changes. The REAME is just there to make sure it provided detailed instructions for dependencies.